### DISCUSSION TITLE
Add some clarity to development guide

### DISCUSSION THREAD

**Discussion by gc-ss** [zitadel/zitadel#2082] [Ideas]
While I can see there's a lot of details at the development guide:

https://github.com/caos/zitadel/blob/main/guides/development.md

… I'm finding it confusing to read as it seems like the concepts are not clearly outlined and structured.

I think https://github.com/caos/zitadel/blob/main/guides/development.md needs to be broken down in more sections and have clearer outlines

**Comment by fforootd** [comment]
Happy to improve up on this basis. 

cc @adlerhurst
